doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -2324,6 +2324,38 @@ The following table indicates the software maturity levels of the support for ea
          - Supported
          - Supported
          - Supported
+       * - **multiple signature keys**
+         - --
+         - --
+         - --
+         - --
+         - --
+         - --
+         - --
+         - --
+         - Experimental
+         - Experimental
+         - Supported
+         - --
+         - --
+         - --
+         - --
+       * - **image encryption**
```

Comment:
```suggestion
       * - **Image encryption**
```
"application image compression" and "hardware cryptography acceleration" could as well be updated as "Application image compression" and "Hardware cryptography acceleration", respectively for consistency.